src/Transaction.proto

```diff
@@ -70,18 +81,6 @@ message CFTransactionEntry {
     bytes PedersenCommit = 2;
```

Comment:
The range proof could go in the CFTransactionEntry

Reply:
Agreed

Reply:
Is there a reason to leave the enum as 8?